ci/cpu/prebuild.sh

```diff
@@ -13,3 +13,10 @@ if [[ "$PYTHON" == "3.7" ]]; then
 else
     export UPLOAD_LIBCUDF=0
 fi
+
+#We only want to upload libcudf_kafka once per python/CUDA combo
+if [ "$PYTHON" == "3.7" && "$CUDA" == "10.1" ]; then
+    export UPLOAD_KAFKA=1
+else
+    export UPLOAD_KAFKA=0
+fi
```

Comment:
Are the kafka packages both independent of python and cuda version?

Reply:
yes, the idea was we just wanted to upload them a single time per build